functions.php

```diff
@@ -260,10 +260,16 @@ public function add_to_context( $context ) {
 		$context['sort_options'] = $this->sort_options;
 		$context['default_sort'] = P4_Search::DEFAULT_SORT;
 
-		$options                          = get_option( 'planet4_options' );
-		$context['donatelink']            = $options['donate_button'] ?? '#';
-		$context['google_tag_value']      = $options['google_tag_manager_identifier'] ?? '';
-		$context['website_navbar_title']  = $options['website_navigation_title'] ?? __( 'International (English)', 'planet4-master-theme' );
+
+		// Do not embed google tag manager js if 'greenpeace' cookie is not set.
+		$cookie_consent = isset( $_COOKIE['greenpeace'] );
+		$gtm            = $options['google_tag_manager_identifier'] ?? '';
+
+		$options                         = get_option( 'planet4_options' );
+		$context['donatelink']           = $options['donate_button'] ?? '#';
+		$context['google_tag_value']     = ! empty( $gtm ) && $cookie_consent ? $gtm : '';
+		$context['google_tag_value']     = $options['google_tag_manager_identifier'] ?? '';
```

Comment:
We don't need `$context['google_tag_value']     = $options['google_tag_manager_identifier'] ?? '';` this line. right?

Reply:
Yeap, you are right. I'll remove it.